app/models/duplicate_match.rb

```diff
@@ -1,10 +1,11 @@
-class FraudMatch < ApplicationRecord
+class DuplicateMatch < ApplicationRecord
+  self.table_name = 'fraud_matches'
```

Comment:
Is the plan to rename the table in a future PR, or leave it as it is?

Reply:
Leave as it is. Otherwise will be a massive work just to change a one-liner.

Reply:
Agreed!

Reply:
Maybe add a comment?

Reply:
Yeah I was just curious! 😄